src/components/UserDashboard/SavedQueries/index.js

```diff
@@ -44,7 +50,8 @@ export const MySavedQueries = compose(
     api,
     theme,
   }) => {
-    const Header = <CardHeader title="Saved Queries" badge={queries.length} />;
+    const savedQueries = queries.filter(isSavedQuery);
```

Comment:
Should this filtering be done in the service that fetches the queries? The component doesn't want to own the logic of separating queries from v-studies

Reply:
Just looked at the state provider, it doesnt separate the fetching into a Riff service file. So can just move the filtering there.......

Reply:
great call! :) thanks!